Makefile

```diff
@@ -15,7 +15,7 @@ T4_OUTPUT_FILES := $(T4_FILES:.tt=.cs)
 
 MONO    := mono
 MSBUILD := msbuild /p:Configuration=$(CONFIG)
-NUNIT := $(MONO) packages/NUnit.ConsoleRunner.3.9.0/tools/nunit3-console.exe
+NUNIT := dotnet test
```

Comment:
Moved to standard dotnet testing tool allowing us to run tests using multiple testing frameworks (like NUnit, xUnit, MSTest) and not just NUnit like what was before. Also the testing was failing as the nunit3-console.exe file was not found in the specified path but this change allowed us to remove third party dependency.